unit_tests/src/test_client_connection.cpp

```diff
@@ -300,3 +300,40 @@ TEST_CASE("#7192: all connections store unique session signatures", "client_conn
 
     CHECK(std::all_of(std::begin(pairs), std::end(pairs), has_unique_session_signatures));
 }
+
+TEST_CASE("#7576: ensure no limitaitons on proxied groupadmin", "client_connection")
```

Comment:
This unit test file isn't the best fit for the test. test_user_administration.cpp is a much better fit since the bug is in that library.

Reply:
Done.